stable/loki/README.md

```diff
@@ -0,0 +1,84 @@
+| `loki.image.pullPolicy`                                   | Image pull policy                             | `Always`                                |
+| `loki.service.port`                                       | Kubernetes port where service is exposed      | `3100`                                  |
+| `loki.service.annotations`                                | Service annotations                           | `{}`                                    |
+| `loki.service.labels`                                     | Custom labels                                 | `{}`                                    |
+| `loki.readinessProbe`                                     | Rediness Probe settings                       | `{}`                                    |
+| `loki.livenessProbe`                                      | Liveness Probe settings                       | `{}`                                    |
+| `loki.resources`                                          | CPU/Memory resource requests/limits           | `{}`                                    |
+| `loki.nodeSelector`                                       | Node labels for pod assignment                | `{}`                                    |
+| `loki.tolerations`                                        | Toleration labels for pod assignment          | `{}`                                    |
+| `loki.affinity`                                           | Affinity settings for pod assignment          | `[]`                                    |
+| `loki.persistence.enabled`                                | Use persistent volume to store data           | `false`                                 |
+| `loki.persistence.accessModes`                            | Persistence access modes                      | `[ReadWriteOnce]`                       |
+| `loki.persistence.storageClassName`                       | Type of persistent volume claim               | `default`                               |
+| `loki.config.auth_enabled`                                | Enable authorization                          | `false`                                 |
+| `loki.config.ingester.lifecycler.ring.store`              | The store way of lifecycler                   | `inmemory`                              |
+| `loki.config.ingester.lifecycler.ring.replication_factor` | The number of lifecycler ring                 | `1`                                     |
+| `loki.schema_configs`                                     | Loki scheam config                            | `[{"from":0,"store":"boltdb","object_store":"filesystem","schema":"v9","index":{"prefix":"index_","period":"168h"}}]` |
+| `loki.storage_configs`                                    | Loki storage config                           | `[{"name":"boltdb","directory":"/data/loki/index"},{"name":"filesystem","directory":"/data/loki/chunks"}]` |
+
+### Promtail
```

Comment:
I'd recommend making promtail/loki either subcharts with a condition making them enable/disablable or wrapping them in conditional logic.

You could easily want to deploy loki on one cluster and have multiple clusters launching promtail and pushing to loki. 
Both should be independently selectable.